proto/cmp/services/book/v1alpha/mint.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package cmp.services.book.v1alpha;
+
+import "cmp/types/v1alpha/common.proto";
+import "cmp/types/v1alpha/currency.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/payment.proto";
+import "cmp/types/v1alpha/pubkey.proto";
+import "cmp/types/v1alpha/traveller.proto";
+import "google/protobuf/timestamp.proto";
+
+message MintRequest {
+  // Message header
+  cmp.types.v1alpha.Header header = 1;
+
+  string validation_id = 2;
+
+  string external_session_id = 3;
+
+  cmp.types.v1alpha.Currency currency = 4;
+
+  cmp.types.v1alpha.Language language = 5;
+
+  string market = 6;
+
+  string booking_reference = 7;
+
+  repeated cmp.types.v1alpha.ExtensiveTraveller travellers = 8;
+
+  string comment = 9;
+
+  // Public keys that will be used to encrypt the private booking data
+  repeated cmp.types.v1alpha.PublicKey public_keys = 10;
+
+  // Payment type, one of "on_chain", "off_chain_invoice", and "off_chain_vcc".
+  cmp.types.v1alpha.Payment payment = 11;
```

Comment:
Same as above with the currency. Shouldn't that be already clear from the search request which currency and subsequently which payment method is being used? Because tbh if I would implement that as a supplier the prices would be different based on the payment method which must be already reflected in the search results.